packages/admin/admin-theme/src/componentsTheme/MuiDataGrid.tsx

```diff
@@ -42,6 +30,8 @@ export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component, {
         root: {
             backgroundColor: "white",
         },
+        /*
+        TODO: @ricky any clue what this is?
```

Comment:
@jamesricky  any clue why we needed this? Its not compatible with the latest version of MUI - Maybe we find some time today to look into this.

Reply:
The element `columnsPanelRow` seems to have been removed in v7.

**[V6 Docs](https://v6.mui.com/x/api/data-grid/data-grid/#data-grid-classes-columnsPanel)**
<img width="879" alt="Screenshot 2024-11-28 at 12 08 27" src="https://github.com/user-attachments/assets/d5091b2f-9617-4c22-93c9-eb4642c34d00">

**[V7 Docs](https://mui.com/x/api/data-grid/data-grid/#DataGrid-css-columnsPanel)**
<img width="881" alt="Screenshot 2024-11-28 at 12 08 08" src="https://github.com/user-attachments/assets/269f12da-ac58-4bbe-b974-f7b0c776fd18">

Maybe you can just select a child element of `columnsPanel` instead, if only the class-key was removed. 
Otherwise we can just check what (if any) styling is now broken and then fix it and remove this part.

Reply:
like we discussed, removed it now